README.md

```diff
@@ -1 +1,2 @@
-# GitDummy
\ No newline at end of file
+# GitDummy
```

Comment:
Please fix this.

Reply:
it is not my problem!

Reply:
Due to inappropriate attitude and behavior, this internship has been concluded as a total failure and a complete fiasca. Your return offer has now been revoked. Please leave the office asap.